sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -108,3 +108,10 @@
 --  #IfDocumentNamingNeeded
 --    desc: populate name field with document names.
 --    arguments: none
+#IfMissingColumn insurance_companies uuid
+ALTER TABLE `insurance_companies` ADD `uuid` binary(16) DEFAULT NULL;
+#EndIf
```

Comment:
When adding a uuid in the upgrade, it should folllow the three steps (showing example of prior entry for users uuid):
```
#IfMissingColumn users uuid
ALTER TABLE `users` ADD `uuid` binary(16) DEFAULT NULL;
#EndIf

#IfUuidNeedUpdate users
#EndIf

#IfNotIndex users uuid
CREATE UNIQUE INDEX `uuid` ON `users` (`uuid`);
#EndIf
```

Reply:
steps:
1. add column
2. populate uuids (this is just to avoid a super long uuid population when a user first uses the api call)
3. add index

Reply:
Done